src/test/groovy/org/powerflows/dmn/engine/model/evaluation/result/DecisionResultSpec.groovy

```diff
@@ -23,14 +23,14 @@ class DecisionResultSpec extends Specification {
 
     void 'should build decision result and be only collection rules result'() {
         given:
-        final String someRule1Entry1Name = 'Some Rule 1 Entry 1 Name'
+        final String someRule1Entry1Name = 'Some YamlRule 1 Entry 1 Name'
```

Comment:
Why YamlRule? This test is a generic. Now suggests is dedicated for Yaml.

Reply:
Idea did this :dagger: